mkosi/distributions/amazonlinux.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from collections.abc import Iterable
+
+from mkosi.config import Architecture
+from mkosi.context import Context
+from mkosi.distributions import fedora, join_mirror
+from mkosi.installer.dnf import Dnf
+from mkosi.installer.rpm import RpmRepository, find_rpm_gpgkey, setup_rpm
+from mkosi.log import die
+
+
+class Installer(fedora.Installer):
```

Comment:
So, since this is something of a Fedora spin-off, should this get a Fedora tools tree by default, like CentOS or Ubuntu? If not, why not?